server/initializers/constants.ts

```diff
@@ -211,7 +217,8 @@ const SCHEDULER_INTERVALS_MS = {
   autoFollowIndexInstances: 60000 * 60 * 24, // 1 day
   removeOldViews: 60000 * 60 * 24, // 1 day
   removeOldHistory: 60000 * 60 * 24, // 1 day
-  updateInboxStats: 1000 * 60// 1 minute
+  updateInboxStats: 1000 * 60, // 1 minute
+  removeTmpVideo: 60000 * 60 // 1 hour
```

Comment:
Either use a scheduler class or a repeatable job, but don't mix them. The scheduler is more simple but don't know when it will run (depends when the server was started)